recipes/ckanweb-deploy-exts.rb

```diff
@@ -221,6 +221,20 @@
 		end
 	end
 
+	if "#{pluginname}".eql? 'data-qld'
```

Comment:
There is no logic to remove this config if the plugin is later removed?

Reply:
original code did not have it either, it was just moved to ensure it did not crash new server builds.